server/pools/tests.py

```diff
@@ -20,8 +21,9 @@ def setUp(self):
                                    end_datetime=datetime(2018, 4, 1, 12, 45))
 
     def test_already_reserved_slots(self):
-        self.assertEqual(self.pool.calculate_already_reserved_slots(datetime(2018, 4, 1, 11, 00), datetime(2018, 4, 1, 12, 30)),
-                         5)
+        self.assertEqual(
+            self.pool.calculate_already_reserved_slots(datetime(2018, 4, 1, 11, 00), datetime(2018, 4, 1, 12, 30)),
+            5)
 
     def test_can_place_reservation(self):
```

Comment:
Split into two separate tests? Also it might need more testing after wrong calculation algorithm is fixed.

Reply:
Yes, but that'll need to be done after that.